deps.edn

```diff
@@ -21,10 +21,14 @@
            ;; linting
            :lint-cache {:replace-paths ["src"]} ;; when building classpath we want to exclude resources
                                                 ;; so we do not pick up our own clj-kondo config exports
-           :clj-kondo {:extra-deps {clj-kondo/clj-kondo {:mvn/version "2021.04.23"}}
+           :clj-kondo {:extra-deps {clj-kondo/clj-kondo {:mvn/version "2021.06.18"}}
                        :main-opts ["-m" "clj-kondo.main"]}
 
-
+           :eastwood {:extra-deps {jonase/eastwood {:mvn/version "0.6.0"}}
+                      :main-opts ["-m" "eastwood.lint" {:source-paths ["src"]
+                                                        :test-paths ["test"]
+                                                        :ignored-faults {:deprecations {rewrite-clj.regression-test true
```

Comment:
These namespaces also report deprecation warnings in the existing `bb ci-unit-tests` step so I figured they're good to ignore.

Reply:
Yeah, this seems good.
There are some tests in these files that exercise deprecated rewrite-clj API fns.